source/plugin/commands/flags.rst

```diff
@@ -0,0 +1,119 @@
+=============
+Command Flags
+=============
+
+Command flags are useful for specifying extra parameters to be used for the processing of a command that doesn't belong
+as a command argument.
+
+To create a flag, we first need a builder for flags. We can simply use the ``flags()`` method provided by
+``GenericArguments`` to obtain the builder we need. From there, we can specify what type of flag we would like to
+create. Note that flags are specified as an argument.
+
+.. code-block:: java
+    
+    import org.spongepowered.api.command.args.GenericArguments;
+    
+    .arguments(GenericArguments.flags().flag("s").buildWith(GenericArguments.none()))
+
+This will create a command flag, so that when the player performs ``/our-command -s``, the flag for ``s`` will be true.
+Note the ``GenericArguments.none()``. This will prevent the command from having any arguments. If you wish for the
```

Comment:
Combine these sentences please - "Note that `GenericArguments.none()` will prevent..."